client/src/languagePlugins/javascript/javascriptPlugin.ts

```diff
@@ -175,31 +175,28 @@ class JavascriptBlockKind implements Langs.Block {
     editor: javascriptEditor,
     evaluate: async (node:Graph.Node) : Promise<Langs.Value> => {
       let jsnode = <Graph.JsNode>node
-      
-      async function getValue(blob) : Promise<Langs.Value> {
-        var pathname = new URL(blob).pathname;
+
+      async function putValue(variableName, hash, value) : Promise<string> {
+        let url = DATASTORE_URI.concat("/"+hash).concat("/"+variableName)
         let headers = {'Content-Type': 'application/json'}
-        let url = DATASTORE_URI.concat(pathname)
         try {
-          const response = await axios.get(url, {headers: headers});
-          console.log(response)
-          return response.data
+          var response = await axios.put(url, value, {headers: headers});
+          // return DATASTORE_URI.concat("/"+hash).concat("/"+variableName)
+          return "http://wrattler_wrattler_data_store_1:7102".concat("/"+hash).concat("/"+variableName)
```

Comment:
We need to figure out how to replace this with some sort of configuration. More comments below.